### PR TITLE
[3.1] Add plugin paste from word to tinymce5

### PR DESCRIPTION
**Pull request by phuongnq**
### Ticket reference or full description of what's in the PR
Similar to https://github.com/craftercms/craftercms/issues/4035, but for 3.1.x

### CODE REVIEW

File: static-assets/js/tinymce-plugins/ace/plugin.min.js [hunk 1]
```diff
@@ -1 +1 @@
...
...-;){let s=(e=t[n]).firstChild,o=!0;for(;s;){if("br"!==s.name.toLowerCase()){o=!1;break}s=s.next}o&&e.unwrap()}});const r=o.parse(e);return e=(new tinymce.html.Serializer).serialize(r)})(s,o),s.execCommand("mceInsertContent",!1,o)};tinymce.PluginManager.add("paste_from_word",function(t,e){let n=!1;t.ui.registry.addToggleButton("paste_as_text",{active:!1,icon:"paste-text",tooltip:"Paste as text",onAction:function(e){n=!n,e.setActive(n),t.focus()},onSetup:function(e){t.on("PastePlainTextToggle",function(n){e.setActive(n.state),t.focus()})}}),t.ui.registry.addToggleMenuItem("paste_as_text",{text:"Paste as text",icon:"paste-text",tooltip:"Paste as text",onAction:function(e){n=!n,t.fire("PastePlainTextToggle",{state:n})},onSetup:function(t){return t.setActive(n),function(){}}}),t.on("paste",function(e){e.preventDefault();const r=((t,e)=>(t=>{const e={};if(t){if(t.getData){const n=t.getData("Text");n&&n.length>0&&-1===n.indexOf("data:text/mce-internal,")&&(e["text/plain"]=n)}if(t.types)for(let n=0;n<t.types.length;n++){const s=t.types[n];try{e[s]=t.getData(s)}catch(t){e[s]=""}}}return e})(e.clipboardData||t.getDoc().dataTransfer))(t,e),l=void 0!==r["text/html"];!n&&l?o(r,t):s(r,t)})})}();
```
Comment (rart) [inline static-assets/js/tinymce-plugins/ace/plugin.min.js]: The PR is missing the changes in the source/unminified version (i.e. `static-assets/js/tinymce-plugins/ace/plugin.js`) from which we build this minified file.

**Review by rart — CHANGES_REQUESTED** [review]

File: static-assets/js/tinymce-plugins/ace/plugin.min.js (hunk 1 above)
Comment (phuongnq) [reply static-assets/js/tinymce-plugins/ace/plugin.min.js]: My bad. I reverted `ace/plugin.js`

File: static-assets/components/cstudio-forms/controls/rte-tinymce5.js [hunk 2]
```diff
@@ -388,7 +396,8 @@ CStudioAuthoring.Module.requireModule(
             contextmenu: !this.enableSpellCheck,
 
             menu: {
-              tools: { title: 'Tools', items: 'tinymcespellchecker code acecode wordcount' }
+              tools: { title: 'Tools', items: 'tinymcespellchecker code acecode wordcount' },
+              edit: { title: 'Edit', items: 'undo redo | cut copy paste paste_as_text | selectall | searchreplace' }
```
Comment (rart) [inline static-assets/components/cstudio-forms/controls/rte-tinymce5.js:400]: Should `paste_as_text` read `paste_from_word` for this to work? Looks like we updated the plugin name but not the button?
```suggestion
              edit: { title: 'Edit', items: 'undo redo | cut copy paste paste_from_word | selectall | searchreplace' }
```

**Review by rart — CHANGES_REQUESTED** [review]

File: static-assets/components/cstudio-forms/controls/rte-tinymce5.js (hunk 2 above)
Comment (phuongnq) [reply static-assets/components/cstudio-forms/controls/rte-tinymce5.js:400]: `paste_as_text` is a button to paste from a document as plain text. In the plugin we include this:

```
    editor.ui.registry.addToggleButton(PLUGIN_PASTE_TEXT_LABEL, {
      active: false,
      icon: PASTE_TEXT_ICON,
      tooltip: PASTE_TEXT_TOOLTIP,
      onAction: function (api) {
        pasteAsTextOption = !pasteAsTextOption;
        api.setActive(pasteAsTextOption);
        editor.focus();
      },
      onSetup: function (api) {
        editor.on('PastePlainTextToggle', function (e) {
          api.setActive(e.state);
          editor.focus();
        });
      }
    });
```
The name can be anything, but my option is that `paste_as_text` is more explained what the button is about.

**Review by rart — APPROVED** [review]

**Pull request merged**